_data/simple-icons.json

```diff
@@ -10226,17 +10232,17 @@
         {
             "title": "X.Org",
             "hex": "F28834",
-            "source": "https://upload.wikimedia.org/wikipedia/commons/9/90/X.Org_Logo.svg"
+            "source": "https://en.wikipedia.org/wiki/File:X.Org_Logo.svg"
         },
         {
             "title": "Xamarin",
             "hex": "3498DB",
-            "source": "https://github.com/dotnet/swag/tree/master/xamarin"
+            "source": "https://github.com/dotnet-foundation/swag/blob/main/xamarin/Xamarin/svg/Xamarin%20logo.svg"
         },
         {
             "title": "XAML",
             "hex": "0C54C2",
-            "source": "https://github.com/microsoft/microsoft-ui-xaml/issues/1185#issuecomment-529731046"
+            "source": "https://microsoft.github.io/microsoft-ui-xaml/"
```

Comment:
I would prefer the original source for this one. There are no vector versions available at this new source.